### PR TITLE
Add missing header options on mobile

### PR DESCRIPTION
**Pull request by SergioEstevao**
## Description

This PR just adds the missing Header options/sizes to the RN mobile app.

## How has this been tested?

This can be tested using this PR on GB-mobile: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1377

## Screenshots 

Simulator Screen Shot - iPhone 11 - 2019-09-23 at 22 08 31

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate.

### CODE REVIEW

**Review by etoledom — APPROVED** [review]
LGTM! 🎉 
Reviewed via https://github.com/wordpress-mobile/gutenberg-mobile/pull/1377#pullrequestreview-293018913

Let's wait for @iamthomasbishop's review before merging 👍

**Comment by iamthomasbishop** [comment]
Looks good! As mentioned in the other thread, I'd like to consolidate these into a single dropdown button, but that's a separate project :)

**Pull request merged**
